tests/Aspire.Dashboard.Components.Tests/Shared/IntegrationTestHelpers.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
+using Xunit;
```

Comment:
Instead of adding an explicit using, please add an implicit one instead (like you did in #8310).

Reply:
@RussKie I think that's better done as a separate change as the analyzer for unused usings is enabled in this repo and there are many existing `using Xunit;`. The goal of #8310 was to minimize the diff when working on this PR.